core/fuzz/utils.rs

```diff
@@ -20,6 +20,19 @@ use std::env;
 use opendal::Operator;
 use opendal::Scheme;
 
+pub fn apply_args_to_env() {
+    // for oss-fuzz integration, set envs from args
+    // example: -opendal_fs_test=on -> OPENDAL_FS_TEST=on
```

Comment:
Oh, I just know that `*.options` work in this way which I don't like.

How about add `dotenvy::from_filename_override(".fs.env")` so that we only need to maintain service related envs?

Reply:
I think it is a better way.

Reply:
However, we split `fuzz_reader` to `fuzz_reader_fs` and `fuzz_reader_mem` after build process.

We seem cannot determine which one we should use in `fuzz_reader.rs`
 
`dotenvy::from_filename_override(".fs.env")` 
`dotenvy::from_filename_override(".mem.env")`

Reply:
We can add in `fn service(scheme: Scheme)` as the first line.

Reply:
For example, `dotenvy::from_filename_override(&format!(".{scheme}.env"))`

Reply:
It actually works but it will init fs and mem services at the same time.

Don't we want to split `fuzz_reader` into `fuzz_reader_fs` and `fuzz_reader_mem`?

If we do that, `fuzz_reader_fs` will init both fs and mem services.

We just do a `cp` operation to split.  `fuzz_reader_fs` and `fuzz_reader_mem` are same.

Reply:
Ok, we can check if `argv[0]` ends with `scheme` and only load related env.

Reply:
Make sense. Thanks for your help！